### PR TITLE
Regenerate billing, core, scheduler, org structs and add source proto files

### PR DESCRIPTION
**Pull request by MuweiHe**
Even though the structs definitions stay all the same, their annotation in protobuf and json handling was causing issues like making the service look for structs in the wrong places. Unify the package and import names in those places to issues of: `Error: unknown value "AUTH_TYPE_LOCAL" for enum [kafka.org](http://kafka.org/).v1.AuthType`. As there were 2 definitions of auth type, one in org structs and one in ccloud-sdk-v2/iam. Change the annotation so service will look for `auth type` in this repo instead of `kafka/org/v1`.

Before:
```
protobuf:"varint,21,opt,name=auth_type,json=authType,proto3,enum=kafka.org.v1.AuthType" json:"auth_type,omitempty" db:"auth_type,omitempty" url:"auth_type,omitempty"
```

After:

```
protobuf:"varint,21,opt,name=auth_type,json=authType,proto3,enum=ccloud.org.AuthType" json:"auth_type,omitempty" db:"auth_type,omitempty" url:"auth_type,omitempty"
```

### CODE REVIEW

**Review by brianstrauch — COMMENTED** [review]
Why is it necessary to add the `vendor` directory?

**Comment by MuweiHe** [comment]
> Why is it necessary to add the `vendor` directory?

for protobuf to use the imports, yes.

**Comment by MuweiHe** [comment]
removed vendor folder. if we ever need to regenerate pb.go files, run `go mod vendor` again.

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,215 @@
+ifeq ($(shell uname),Darwin)
```
Comment (brianstrauch) [inline Makefile:1]: Do we need to add this Makefile?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+CLEAN_TARGETS += protoc-clean prototool-clean
+GO_EXTRA_DEPS := install-protoc protoc-binaries install-mocker install-hub install-prototool install-protoc-gen-validate
+TEST_TARGETS += mods-vet
+GO_TEST_TARGET := lint-go run-go-tests
+BUILD_GO_OVERRIDE := generate-mocks compile
+GO111MODULE := on
+GO_USE_VENDOR ?= -mod=vendor
+export GO111MODULE
+
+include ./mk-include/cc-begin.mk
```
Comment (brianstrauch) [inline Makefile:16]: Looks like these haven't been added. They're not typically added to public repositories in the first place.

File: go.mod [hunk 3]
```diff
@@ -1,25 +1,30 @@
 module github.com/confluentinc/ccloud-sdk-go-v1-public
 
 require (
-	github.com/confluentinc/proto-go-setter v0.0.0-20201026155413-c6ceb267ee65
-	github.com/dghubble/sling v1.2.1-0.20181125223409-7458fd7fa70b
-	github.com/envoyproxy/protoc-gen-validate v0.6.7
+	github.com/confluentinc/cc-structs/operator v0.1543.0
```
Comment (brianstrauch) [inline go.mod:4]: We shouldn't be adding private repositories

File: tool.go [hunk 4]
```diff
@@ -0,0 +1,17 @@
+//go:build tools
```
Comment (brianstrauch) [inline tool.go:1]: Rename this file to tools.go?

**Review by brianstrauch — CHANGES_REQUESTED** [review]

File: go.mod (hunk 3 above)
Comment (MuweiHe) [reply go.mod]: it was in tools.go. removed it!

File: Makefile (hunk 1 above)
Comment (MuweiHe) [reply Makefile]: just one target. Removed all others!

**Review by brianstrauch — APPROVED** [review]
Looks good! Can you delete the .DS_Store file?

**Pull request merged**